frontend/src/model/beta/releases/ModelReleaseReviewBanner.tsx

```diff
@@ -3,26 +3,29 @@ import Button from '@mui/material/Button'
 import Typography from '@mui/material/Grid'
 import Paper from '@mui/material/Paper'
 import { useTheme } from '@mui/material/styles'
-import { useGetReleasesForModelId } from 'actions/release'
 import { useState } from 'react'
 import { mutate } from 'swr'
 
+import { useGetReleasesForModelId } from '../../../../actions/release'
+import { useGetReviewRequestsForModel } from '../../../../actions/review'
 import { postReviewResponse } from '../../../../actions/review'
 import { ReleaseInterface } from '../../../../types/types'
 import ReviewWithComment, { ResponseTypeKeys } from '../../../common/ReviewWithComment'
 
 type ModelReleaseReviewBannerProps = {
   label: string
   release: ReleaseInterface
-  mutateReviews: () => void
 }
 
-export default function ModelReleaseReviewBanner({ label, release, mutateReviews }: ModelReleaseReviewBannerProps) {
+export default function ModelReleaseReviewBanner({ label, release }: ModelReleaseReviewBannerProps) {
   const theme = useTheme()
 
   const { mutateReleases } = useGetReleasesForModelId(release.modelId)
+  const { mutateReviews: mutateActiveReviews } = useGetReviewRequestsForModel(release.modelId, release.semver, true)
+  const { mutateReviews: mutateInactiveReviews } = useGetReviewRequestsForModel(release.modelId, release.semver, true)
```

Comment:
the active param should be `false` here: `useGetReviewRequestsForModel(release.modelId, release.semver, false)`